akka-projection-eventsourced/src/main/scala/akka/projection/eventsourced/scaladsl/EventSourcedProvider.scala

```diff
@@ -14,7 +14,7 @@ import akka.persistence.query.NoOffset
 import akka.persistence.query.Offset
 import akka.persistence.query.PersistenceQuery
 import akka.persistence.query.scaladsl.EventsByTagQuery
-import akka.projection.eventsourced.EventEnvelope
+import akka.projection.EventEnvelope
```

Comment:
I don't like the move of EventEnvelope into core. It's specific to akka-projection-eventsourced. To work around the cyclic dependencies the tests can be placed in a separate projection, akka-projection-eventsourced-tests.